src/home_robot_hw/home_robot_hw/nodes/goto_controller.py

```diff
@@ -132,10 +132,11 @@ def _run_control_loop(self):
         while not rospy.is_shutdown():
             if self.active and self.xyt_goal is not None:
                 # Compute control
-                v_cmd, w_cmd = self.controller.compute_control()
+                v_cmd, w_cmd, done = self.controller.compute_control()
```

Comment:
`self.controller` is not `DiffDriveVelocityController` (the one you made modifications to) but `GotoVelocityController`. You might need to make changes there too.